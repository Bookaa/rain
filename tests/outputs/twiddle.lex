keyword 'import'
name 'bits'
newline
keyword 'let'
name 'main'
symbol '='
keyword 'func'
symbol '('
symbol ')'
indent
keyword 'let'
name 'a'
symbol '='
int 38
newline
keyword 'let'
name 'b'
symbol '='
int 3
newline
name 'print'
symbol '('
name 'bits'
symbol '.'
name 'lsh'
symbol '('
name 'a'
symbol ','
name 'b'
symbol ')'
symbol ')'
newline
name 'print'
symbol '('
name 'bits'
symbol '.'
name 'rsh'
symbol '('
name 'a'
symbol ','
name 'b'
symbol ')'
symbol ')'
newline
name 'print'
symbol '('
name 'bits'
symbol '.'
name 'and'
symbol '('
name 'a'
symbol ','
name 'b'
symbol ')'
symbol ')'
newline
name 'print'
symbol '('
name 'bits'
symbol '.'
name 'or'
symbol '('
name 'a'
symbol ','
name 'b'
symbol ')'
symbol ')'
newline
name 'print'
symbol '('
name 'bits'
symbol '.'
name 'xor'
symbol '('
name 'a'
symbol ','
name 'b'
symbol ')'
symbol ')'
newline
name 'print'
symbol '('
name 'bits'
symbol '.'
name 'not'
symbol '('
name 'a'
symbol ')'
symbol ')'
newline
dedent
newline
EOF
