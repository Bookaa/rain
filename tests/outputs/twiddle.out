304
4
2
39
37
-39
